test/performance/threshold

```diff
@@ -0,0 +1 @@
+total	average	750
```

Comment:
Can this file have a new line please?

Reply:
It's very fussy when it comes to formatting, but will give it a go.